shared/macros-bash.jinja

```diff
@@ -107,6 +107,55 @@ if test -x /usr/sbin/prelink; then
 fi
 {{%- endmacro -%}}
 
+{{#
+# Macro to configure DConf settings for RHEL and Fedora systems.
+#}}
+{{%- macro dconf_settings(path, key, value, db, setting_file) -%}}
+# Check for setting in any of the DConf db directories
+# If files contain ibus or distro, ignore them.
+# The assignment assumes that individual filenames don't contain :
+readarray -t SETTINGSFILES < <(grep -r "\\[{{{ path }}}\\]" "/etc/dconf/db/" | grep -v "distro\\|ibus" | cut -d":" -f1)
```

Comment:
When you migrate from double quotes to single quotes in bash, double backslashes inside those double quotes can be converted to single backslashes (in single quotes). However, this line contains the `path` macro expansion which can be a bash variable, so quotes should remain double. But `"distro\\|ibus"` can be converted to `'distro\|ibus'` and so on.